pkgs/deb/debian/libxnvme-dev.install

```diff
@@ -0,0 +1,4 @@
+usr/include/libxnvme*.h
+usr/lib/*/libxnvme.a
+usr/lib/*/libxnvme.so
```

Comment:
Does this need to be here? I see that libxnvme-dev depends on libxnvme0 - thus libxnvme-dev shouldn't be installing the `.so`

Reply:
Hmm, however, it might be what can enable the use of multiple naming conventions... gonna take it for a spin having "libxnvme.so.0" and "libxnvme.so.0.7.3"